### ISSUE TITLE
What is difference between chat with your own data report under azure

### ISSUE THREAD

**Issue by venkateshkumar** [microsoft/PubSec-Info-Assistant#409]
Dear Team,

Just wondering which solution accelerator we need to use for chatting with our data or this accerlator.
Not seen much differences on the backend
Request you to provide information.

Warm regards,
Venkateshkumar.S

**Comment by ArpitaisAn0maly** [comment]
Hi [venkateshkumar](https://github.com/venkateshkumar). 

- We offer more file type support. 20 different file types including images. 
- You can do image search without OCR. 
- We have true implementation of " bring your own embedding" where you can use open source embedding model or AOAI embedding model. 
- We ahve folders and tagging enabled which allows you to do targeted search on specific version of document or specific subject. 
- We don't use OOTB skills to do index enrichment. we have cutom code for index enrichment. 
- We also allow you configure chat experience with Persona setting where you can configure your bot to act as teacher vs student.

In summary we offer more customization for the features we provide. I hope this answers your question.

**Comment by dayland** [comment]
Marking for closure due to no response in 2 weeks.

**Comment by dayland** [comment]
Closing due to inactivity.
